css-layout-classes/styles.css

```diff
@@ -1,18 +1,17 @@
 .container {
   max-width: 800px;
-  margin: auto;
+  margin-left: auto;
+  margin-right: auto;
 }
 
 .row {
   display: flex;
 }
 
-.fit {
-  display: flex;
-  justify-content: center;
+.column-size {
```

Comment:
Why isn't this the `column-fourth` rule? This is the class setting the size of the element 🤔